.github/workflows/arm-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: ARM tests
+on:
+  push:
+    branches:
+      - master
+      - ci-changes # TODO: remove
+
+jobs:
+  test:
+    runs-on: [self-hosted, Linux, ARM64]
```

Comment:
I'm not finding a clear answer to this in the GHA docs, but where is `self-hosted` configured? Or does our ARM runner poll recent GHA builds and kicks off when it encounters a `self-hosted` one?